SpatialGDK/Source/SpatialGDK/Public/Utils/InterestFactory.h

```diff
@@ -16,6 +16,8 @@ DECLARE_LOG_CATEGORY_EXTERN(LogInterestFactory, Log, All);
 namespace SpatialGDK
 {
 
+void GatherClientInterestDistances();
```

Comment:
What do you think about making this a static public function inside the InterestFactory? Just to show that it's related to it.